htcondor/publishers/publisher.py

```diff
@@ -156,29 +149,35 @@ def dataframe_to_classads(dataframe):
 
     :type dataframe: :obj:`DataFrame`
     """
+    logging.getLogger().debug("converting dataframe to classads")
     ads = []
     records = dataframe.to_dict(orient='records')
-    for record in records:
-        # NOTE: Pandas will add NaN for some of the values. This causes
-        #       extremely undesired/unexpected issues. Better to remove
-        #       NaN values before converting a dataframe row to classad.
-        ad_dict = {}
-        for key, value in record.items():
-            if pandas.isnull(value):
-                continue
-            # The massage below is needed to handle Issue #263
-            # https://github.com/HEPCloud/decisionengine_modules/issues/263
-            # after switch to python3
-            if isinstance(value, bytes):
-                ad_dict[key] = value.decode("latin-1")
-            elif isinstance(value, str):
-                if value.startswith("b'"):
-                    ad_dict[key] = value[2:].rstrip("'")
+
+    try:
+        for record in records:
+            # NOTE: Pandas will add NaN for some of the values. This causes
+            #       extremely undesired/unexpected issues. Better to remove
+            #       NaN values before converting a dataframe row to classad.
+            ad_dict = {}
+            for key, value in record.items():
+                if pandas.isnull(value):
+                    continue
+                # The massage below is needed to handle Issue #263
+                # https://github.com/HEPCloud/decisionengine_modules/issues/263
+                # after switch to python3
+                if isinstance(value, bytes):
+                    ad_dict[key] = value.decode("latin-1")
+                elif isinstance(value, str):
+                    if value.startswith("b'"):
+                        ad_dict[key] = value[2:].rstrip("'")
+                    else:
+                        ad_dict[key] = value
                 else:
                     ad_dict[key] = value
-            else:
-                ad_dict[key] = value
-        ad = classad.ClassAd()
-        ad.update(ad_dict)
-        ads.append(ad)
-    return ads
+            ad = classad.ClassAd()
+            ad.update(ad_dict)
+            ads.append(ad)
+    except Exception:
```

Comment:
since this looks like a defensive `except`, can you add `# pragma: no cover` to the `except Exception:` line?